docs/site/LB3-vs-LB4-request-response-cycle.md

```diff
@@ -0,0 +1,85 @@
+`app.post()`, `app.use()`, etc., just like how you do in Express.
+In LoopBack 4, you cannot do it anymore. The closest to doing this is to use the
+[RestApplication.mountExpressRouter() ](https://loopback.io/doc/en/lb4/apidocs.rest.restapplication.mountexpressrouter.html)
+API. This API allows you to mount a custom Express router on the LoopBack app.
+
+Using [Controllers](https://loopback.io/doc/en/lb4/Controllers.html) is the
+recommended way for creating custom (and REST) endpoints on your app. Its
+support for
+[dependency injection](https://loopback.io/doc/en/lb4/Dependency-injection.html)
+and [Interceptors](https://loopback.io/doc/en/lb4/Interceptors.html) makes it a
+very powerful extension mechanism.
+
+LoopBack 4 has gotten rid of
+[middleware.json](https://loopback.io/doc/en/lb3/middleware.json.html)
+by. It is not require anymore because of the architectural changes.
+
+In LoopBack 3, models files automatically created the corresponding REST API
+endpoints and the database query machinery. In LoopBack, model files are limited
+only to describing the properties of the data. You will have to create a
+corresponding [Repository](https://loopback.io/doc/en/lb4/Repositories.html) for
+database connectivity, and controller creating the REST API endpoint.
+
+The fact that you have to create two more artifacts along with the model to
+get a REST endpoint working might seem overly tedious at first. However, the
+separation of concerns and decoupling the functionality makes the codebase
+cleaner, easier to maintain, and much easier to customize functionality at
+various levels. This can be better appreciated as the complexity of your app
+grows.
+
+Components are still supported in LoopBack 4, but the concept of component
+has completely changed.
+
+In LoopBack 3, a
+[component](https://loopback.io/doc/en/lb3/LoopBack-components.html)
+is a simple Node.js module which exports a function with the signature
+`function(app, options)`. In LoopBack 4, a
+[component](https://loopback.io/doc/en/lb4/Creating-components.html)
+is a TypeScript class which can add
+[servers](https://loopback.io/doc/en/lb4/Server.html),
+[observers](https://loopback.io/doc/en/lb4/Life-cycle.html),
+[providers](https://loopback.io/doc/en/lb4/Creating-components.html#providers),
+and controllers to the app using dependency injection.
+
+A components from LoopBack 3 can be migrated to LoopBack 4 by moving the
+functionality to the controller of a LoopBack 4 component.
+
+Because of the architectural changes, `component-config.json` is not required
+in LoopBack 4 anymore.
+
+If you used LoopBack 3 boot scripts for adding routes to the app, it
+should now be moved to a standalone controller, a component, or implemented
+using `app.mountExpressRouter()`.
+
+{% include tip.html content="For details about migrating LoopBack 3 boot scripts
+refer to
+[Migrating boot scripts](https://loopback.io/doc/en/lb4/migration-boot-scripts.html)
+." %}
+
+### Request/response pathway
```

Comment:
Are the contents of `Request/response pathway` forthcoming` ?

Reply:
Yes :D